internal/commands/ipaddress/show.go

```diff
@@ -47,7 +47,11 @@ func (s *showCommand) Execute(exec commands.Executor, arg string) (output.Output
 					{Title: "Family:", Key: "access", Value: ipAddress.Family},
 					{Title: "Part of Plan:", Key: "access", Value: ipAddress.PartOfPlan, Format: output.BoolFormat},
 					{Title: "PTR Record:", Key: "access", Value: ipAddress.PTRRecord},
-					{Title: "Server UUID:", Key: "access", Value: ipAddress.ServerUUID},
+					{
+						Title: "Server UUID:",
+						Key:   "access", Value: ipAddress.ServerUUID,
```

Comment:
The keys get used in json/yaml output so probably dont want to name them all the same... 

(sidenote: we should go through all of these i guess?)

Reply:
(or; should we validate this even when rendering?)

Reply:
:scream: didn't even noticed it, yes it should be renamed properly